tests/ptvsd/server/test_flask.py

```diff
@@ -65,63 +65,56 @@ def test_flask_breakpoint_no_multiproc(bp_target, start_method):
         session.start_debugging()
 
         # wait for Flask web server to start
-        wait_for_connection(FLASK_PORT)
-        link = FLASK_LINK
-        web_request = get_web_content(link, {})
+        with flask_server:
+            home_request = flask_server.get("/")
 
-        thread_stopped = session.wait_for_next(
-            Event("stopped"), some.dict.containing({"reason": "breakpoint"})
-        )
-        assert thread_stopped.body["threadId"] is not None
-
-        tid = thread_stopped.body["threadId"]
+            thread_stopped = session.wait_for_next(
```

Comment:
`wait_for_stop`

Reply:
(and I think the below can be simplified a lot then? since it's doing what `wait_for_stop` would do, like fetching frames etc)